r/R/arrow-datum.R

```diff
@@ -70,13 +70,50 @@ as.vector.ArrowDatum <- function(x, mode) {
 
 #' @export
 Ops.ArrowDatum <- function(e1, e2) {
```

Comment:
Why the changes to Ops.ArrowDatum? I don't see any test changes, and I don't follow how this change is an improvement.